src/components/combo_box/_combo_box.scss

```diff
@@ -165,6 +165,19 @@
     }
   }
 
+  // Overrides the top and bottom padding of 8px with ouiFormControlLayout--group 
+  // when append/prepend is enabled, original top, bottom padding was 1px
+  .ouiFormControlLayout--group {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
+
+  // Overrides line-height of 16px coming from ouiFormControlLayout--group .ouiText
+  .ouiFormControlLayout--group .ouiText {
+    // sass-lint:disable-block no-important
+    line-height: 0 !important;
```

Comment:
Is it possible to avoid !important with specificity of selectors? Or are there too many to make that worthwhile?

Reply:
Even I wasn't happy on this one, I was banking on specificity as there are added under combo box class but then this was necessary as the one I'm overriding had important added to it.